src/Options.js

```diff
@@ -156,6 +157,7 @@ addFeatureOption('generatorComprehension', EXPERIMENTAL);
 addFeatureOption('memberVariables', EXPERIMENTAL);
 addFeatureOption('require', EXPERIMENTAL);
 addFeatureOption('strongMode', EXPERIMENTAL);
+addFeatureOption('strongModeRuntime', EXPERIMENTAL);
```

Comment:
The intention was to add all features that have a runtime overhead under this option.

@johnjbarton Other things include checking that no holes are created on arrays, making class instances sealed, functions non constructable, writing to non existing properties errors etc. Some of these are assert like and others are making things more locked down.